content/en/releases/version-skew-policy.md

```diff
@@ -217,7 +217,7 @@ Optionally upgrade `kubelet` instances to **{{< skew currentVersion >}}** (or th
 
 {{< note >}}
 Before performing a minor version `kubelet` upgrade, [drain](/docs/tasks/administer-cluster/safely-drain-node/) pods from that node.
-In-place minor version `kubelet` upgrades are not supported.
+In-place minor version `kubelet` upgrades, without a node drain, are not supported.
```

Comment:
We could write:
```suggestion
In-place`kubelet` upgrades to a newer minor version of Kubernetes, without a node drain, are not supported,
and if you do this the node's behavior is undefined.
```

Do you prefer that?

(For clarity about “minor” version upgrades, there is no
skew policy yet for an upgrade to a future release of Kubernetes that has a different major version).

Reply:
I prefer:

`In-place `kubelet` upgrades to a newer minor version of Kubernetes, without a node drain are not supported.`

Just becaus the language sounds tighter and for me the `node's behaviour is undefined` implies that the node might not upgrade properly when I believe it's really about the pods?